src/backend/distributed/commands/type.c

```diff
@@ -712,15 +712,13 @@ RecreateCompositeTypeStmt(Oid typeOid)
 /*
  * attributeFormToColumnDef returns a ColumnDef * describing the field and its property
  * for a pg_attribute entry.
- *
- * Note: Current implementation is only covering the features supported by composite types
```

Comment:
Do we support all the possible attributeForm parameters now? Can you clarify the reason why we remove this note?

Reply:
This note incentivized the reader to ignore typemod being hardcoded as -1